### PR TITLE
Added new collapsible module without state dependency

### PR DESCRIPTION
**Pull request by PrashanthKumarM**

### CODE REVIEW

**Review by Nethracs — COMMENTED** [review]
Change the test comment 'should have the opposite drawerExpanded value when toggled'

File: actionlist/__tests__/actionItemsSpec.js [hunk 1]
```diff
@@ -2,31 +2,16 @@ import deepFreeze from 'deep-freeze';
 import expect from 'expect';
 
 import reducer from '../app/reducers/action_items';
-import { toggleItem } from '../app/actions/action_items';
+import { toggleDrawer } from '../app/actions/action_items';
 
 describe('action_items', () => {
-  it('should have the opposite expanded value when toggled', () => {
-    const cardToExpand = 'foo';
+  it('should have the opposite drawerExpanded value when toggled', () => {
```
Comment (Nethracs) [inline actionlist/__tests__/actionItemsSpec.js:8]: Change the comment for the test

**Review by pandeyh92 — APPROVED** [review]

File: actionlist/app/actions/action_items.js [hunk 2]
```diff
@@ -1,6 +1,8 @@
 import { createAction } from 'redux-actions';
```
Comment (Nethracs) [inline actionlist/app/actions/action_items.js:1]: change file names to camelcase

File: actionlist/app/components/action_item_body.js [hunk 3]
```diff
@@ -40,55 +40,54 @@ const routeLog = () => ({});
```
Comment (Nethracs) [inline actionlist/app/components/action_item_body.js:40]: change file name

File: actionlist/app/components/action_item_header.js [hunk 4]
```diff
@@ -0,0 +1,62 @@
+import React from 'react';
```
Comment (Nethracs) [inline actionlist/app/components/action_item_header.js:1]: File name needs change

File: actionlist/__tests__/actionItemsSpec.js (hunk 1 above)
Comment (PrashanthKumarM) [reply actionlist/__tests__/actionItemsSpec.js:8]: Please check the changes

**Comment by Nethracs** [comment]
changing the file names to camelcase is created as an issue. Approving the pull request.

**Review by Nethracs — APPROVED** [review]

**Pull request merged**